cmd/kube-controller-manager/app/options/options.go

```diff
@@ -235,7 +235,7 @@ func (s *CMServer) AddFlags(fs *pflag.FlagSet, allControllers []string, disabled
 }
 
 // Validate is used to validate the options and config before launching the controller manager
-func (s *CMServer) Validate(allControllers []string, disabledByDefaultControllers []string) error {
```

Comment:
That's odd.  There two in this list that either belonged in a separate controller process or a separately enabled bits since they were deployment specific.